.rubocop.yml

```diff
@@ -23,9 +38,20 @@ Lint/HandleExceptions:
 Lint/LiteralInInterpolation:
   Enabled: false
 
-# This should go down over time.
-MethodLength:
-  Max: 40
+Style/SpaceAroundOperators:
+  MultiSpaceAllowedForOperators:
+      - '='
+      - '||='
+      - '==='
+      - '=>'
+      - '<<'
+
+Style/AccessModifierIndentation:
+  Enabled: false
```

Comment:
if we remove that we get something like this
```
lib/rspec/core/formatters/console_codes.rb:23:9: C: Outdent access modifiers like module_function.
        module_function
        ^^^^^^^^^^^^^^^
```
we would have to outdent `module_function`, are you ok with it?